### DISCUSSION TITLE
"Delete confirmation" how to have it back?

### DISCUSSION THREAD

**Discussion by lokisonline** [zanllp/infinite-image-browsing#472] [General]
Hi all! When i installed this extension itr asked me for delete confirmation, i disabled it from the window, but now i want it back because the delete button is too easy to press. How can i have it back? thx!

**Comment by zanllp** [comment]
<img width="536" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/e919af30-45a7-42c8-b738-e12c06395792">

**Reply by lokisonline** [reply]
Thank you so much!!! :D
